docs/tags.md

```diff
@@ -28,10 +28,9 @@ In our Cypress and Cucumber setup, tags are used to categorize and selectively r
 - **@unstable_test:** These are tests that have been identified as unstable. They may fail intermittently due to issues in either the staging or development environments. This tag helps in identifying flaky tests that require attention.
   <br>
 
-- **@group-A:** This tag is used for grouping the certain feature files into 'group-A' for the parallel tests execution in the CI/Github actions.
-  <br>
+- **@group:** Used to group test files for parallel execution in CI. This helps divide tests across multiple runners for faster test execution.
```

Comment:
missing `-*` so it looks like `@group-*`

Reply:
did in different way, @group-X.